core/src/main/java/com/datastrato/gravitino/Configs.java

```diff
@@ -12,113 +12,120 @@
 import java.util.List;
 import org.apache.commons.lang3.StringUtils;
 
-public interface Configs {
+public class Configs {
```

Comment:
Hi @kiratkumar47 would you please add a private constructor for this class `Configs`?

Reply:
Hi @jerryshao 
It is done.